packages/cms/app/helmet.js

```diff
@@ -1,6 +1,6 @@
 export default {
   link: [
-    {rel: "icon", href: "images/favicon.ico"}
+    {rel: "icon", href: "/images/favicon.ico"}
```

Comment:
Not necessary as far as I know, so I'm removing it. The only reason I even changed it was actually creating a broken request to my custom `/image/` api endpoint, so would rather remove it anyway.

Reply:
This line shouldn't be removed, just the forward-slash that got added should be removed.

Reply:
@davelandry When the forward slash isn't there, a request is made to `[currentpage]/images/favico.ico`.  Because of `routes.jsx`, the main page attempts to load a profile with a slug of `images` and an id of `favico.ico`, resulting in a failed profile lookup on every page load. Do you have a preference on how to handle this?

Reply:
For now, I've added the line back in with no leading slash. Let me know your thoughts on the routing problem.